### ISSUE TITLE
RTSP basic authorization

### ISSUE THREAD

**Issue by madaotw** [mrmaffen/vlc-android-sdk#80]
Hello, I am testing some ONVIF devices with rtsp urls which are like rtsp://[username]:[password]@192.168.x.x:554/cam/realmonitor?channel=1&subtype=0&unicast=true&proto=Onvif, but they can't work. 
I'd like to know how to put header which contains username and password on this project. Thanks!

**Comment by xieyingwu** [comment]
Yep,I also meet the question,I in the close issues found the question like this question but i found the question had in two years to report the VLC,so i wan to ask the mrmaffen ,you can fix the question.

**Comment by xieyingwu** [comment]
@mrmaffen
